index.html

```diff
@@ -5,51 +5,50 @@
-          euismod elit sodales placerat. Proin pretium arcu a odio consectetur,
-          nec vestibulum nisi venenatis. Nulla eleifend nisi vel pretium
-          aliquam. Cras ornare elementum vehicula. Curabitur at hendrerit nisl.
-          Donec ligula mauris, commodo non diam et, aliquam aliquet lectus.
-          Maecenas volutpat, nisl in consectetur condimentum, ligula mauris
-          fringilla mauris, eu aliquet mauris ligula at orci. Cras efficitur non
-          massa in auctor. Sed posuere leo quam, sed varius massa euismod sit
-          amet. Proin vehicula egestas massa. Nullam tempus massa a elementum
-          posuere. Nam ac velit ac purus vestibulum fringilla. Nunc a tortor
-          posuere, ultrices nisi id, interdum risus. Donec laoreet tempus libero
-          ut viverra. Proin dapibus magna vel urna ornare, et pretium metus
-          facilisis.
-        </p>
-        <div class="button">
-          <p>GET STARTED</p>
-        </div>
-        <!-- <button>GET STARTED</button> -->
-    </div>
-    <div class="footer">
-      <p>Copyright by photography123 - all right reserved</p>
-    </div>
+    <main class="sp__content">
+      <img src="img/photography.jpg" alt="Taking Piture from SmartPhone" />
 
+      <h2 class="sp__content-header">
+        Rappresent your life with a simple photo
+      </h2>
+      <p class="sp__paragraph">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam velit
+        est, ullamcorper a vestibulum eu, pulvinar ut orci. Donec gravida orci
+        non suscipit consequat. Suspendisse ut dui sed tortor ultrices tempus.
+        Phasellus dictum lectus ex, ut placerat massa pharetra at. Nullam felis
+        dolor, volutpat nec congue eget, ornare vitae quam. In finibus urna vel
+        molestie tempus. Duis et convallis odio, convallis porta eros. Curabitur
+        vel leo turpis. Morbi consequat ante in odio varius hendrerit.
+        <br /><br />
+        Vestibulum varius pretium mattis. Aliquam vulputate maximus est, quis
+        euismod elit sodales placerat. Proin pretium arcu a odio consectetur,
+        nec vestibulum nisi venenatis. Nulla eleifend nisi vel pretium aliquam.
+        Cras ornare elementum vehicula. Curabitur at hendrerit nisl. Donec
+        ligula mauris, commodo non diam et, aliquam aliquet lectus. Maecenas
+        volutpat, nisl in consectetur condimentum, ligula mauris fringilla
+        mauris, eu aliquet mauris ligula at orci. Cras efficitur non massa in
+        auctor. Sed posuere leo quam, sed varius massa euismod sit amet. Proin
+        vehicula egestas massa. Nullam tempus massa a elementum posuere. Nam ac
+        velit ac purus vestibulum fringilla. Nunc a tortor posuere, ultrices
+        nisi id, interdum risus. Donec laoreet tempus libero ut viverra. Proin
+        dapibus magna vel urna ornare, et pretium metus facilisis.
+      </p>
+      <button class="button">GET STARTED</button>
+    </main>
+    <footer class="footer">
```

Comment:
Better to use the next properties to customize the footer

https://stackblitz.com/edit/js-c6lota?file=index.js,index.html,style.css